rpc/mochostagent/security/certificate/moc_mochostagent_certificate.proto

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache v2.0 license.
+
+syntax = "proto3";
+option go_package = "github.com/microsoft/moc/rpc/mochostagent/security";
+package moc.mochostagent.security;
+import "google/protobuf/wrappers.proto";
+import "moc_common_common.proto";
+
+enum CertificateType {
+	Client = 0;
+	Server = 1;
+}
+
+message CertificateRequest {
+	repeated Certificate Certificates = 1;
+}
+
+message CertificateResponse {
+	repeated Certificate Certificates = 1;
```

Comment:
[nit] indentation

Reply:
Fixed indention and spacing issues.

Conventionally I have noticed the Error is always capitalized so kept the same here for consistency.